### PR TITLE
Updated README to work with OMCompiler beeing no submodule

### PR DESCRIPTION
**Pull request by AnHeuermann**
Fixed old link directing to archived submodule OpenModelica/OMCompiler.

### CODE REVIEW

File: OMCompiler/README.Linux.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ sudo apt-get build-dep openmodelica
 git clone --recursive https://openmodelica.org/git-readonly/OpenModelica.git OpenModelica
 cd OpenModelica
 autoconf
-./configure --with-omniORB
```
Comment (AnHeuermann) [inline OMCompiler/README.Linux.md]: Is this `omniORB` actually the standard way to compile on Linux? I never used it.

Comment (sjoelund) [reply OMCompiler/README.Linux.md]: Yes, you need it if you use the old versions of clients such as OMPython.

Comment (AnHeuermann) [reply OMCompiler/README.Linux.md]: Okay, so the best way would be `./configure CC=clang CXX=clang++ --with-omniORB`? That of course only works if you have clang.
Or I just leaf it as it was.

Comment (AnHeuermann) [reply OMCompiler/README.Linux.md]: @sjoelund I reverted the changes on this line. Do you approve the PR?

**Pull request merged**